### PR TITLE
maven 3.3.1

### PR DESCRIPTION
**Pull request by rlespinasse**
http://maven.apache.org/docs/3.3.1/release-notes.html

### CODE REVIEW

**Comment by asparagui** [comment]
See also #37811.

File: Library/Formula/maven.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Maven < Formula
   homepage "https://maven.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz"
-  mirror "https://archive.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz"
-  sha256 "8c190264bdf591ff9f1268dc0ad940a2726f9e958e367716a09b8aaa7e74a755"
+  url "https://archive.apache.org/dist/maven/maven-3/3.3.1/binaries/apache-maven-3.3.1-bin.tar.gz"
+  mirror "http://apache.mirrors.ovh.net/ftp.apache.org/dist/maven/maven-3/3.3.1/binaries/apache-maven-3.3.1-bin.tar.gz"
```
Comment (mikemcquaid) [inline Library/Formula/maven.rb]: Why the mirror change?

Comment (rlespinasse) [inline Library/Formula/maven.rb]: I thought the url page no longer appropriate. After more checking, I was wrong. I put up the url and the mirror in version 3.3.1.

File: Library/Formula/maven.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Maven < Formula
   homepage "https://maven.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz"
-  mirror "https://archive.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz"
-  sha256 "8c190264bdf591ff9f1268dc0ad940a2726f9e958e367716a09b8aaa7e74a755"
+  url "http://www.apache.org/dyn/closer.cgi?path=maven/maven-3/3.3.1/binaries/apache-maven-3.3.1-bin.tar.gz"
```
Comment (mikemcquaid) [inline Library/Formula/maven.rb]: Sorry, last thing: can this URL be made `https` too? Thanks!

Comment (rlespinasse) [inline Library/Formula/maven.rb]: sure

**Comment by mikemcquaid** [comment]
Thanks @rlespinasse!

**Pull request closed without merging**